agot-bg-game-server/src/client/IngameComponent.tsx

```diff
@@ -602,12 +605,35 @@ export default class IngameComponent extends Component<IngameComponentProps> {
                         <Card.Header>
                             <Nav variant="tabs">
                                 <Nav.Item>
-                                    <Nav.Link eventKey="game-logs">Game Logs</Nav.Link>
+                                    <Nav.Link eventKey="game-logs">
+                                            <OverlayTrigger
+                                                overlay={<Tooltip id="note">Game Logs</Tooltip>}
+                                                placement="top"
+                                            >
+                                            <span>
+                                                <FontAwesomeIcon
+                                                    style={{ color: "white" }}
+                                                icon={faHistory} />
+                                                {/* &nbsp;Logs */}
+                                            </span>
+                                            </OverlayTrigger>
+                                        </Nav.Link>
                                 </Nav.Item>
                                 <Nav.Item>
                                     <div className={classNames({ "new-event": this.publicChatRoom.areThereNewMessage })}>
                                         <Nav.Link eventKey="chat">
-                                            Chat
+                                            <OverlayTrigger
+                                                overlay={<Tooltip id="note">Game Chat</Tooltip>}
```

Comment:
LGTM so far. But here I see all tooltips share the same element Id. Can you change them so they match to the tooltip content, sth like `note-tooltip`, `settings-tooltip`, etc.. in addition there formatting (indentation) could be better, e.g. closing navlink Tag is not on same indent as its opening Tag…

Reply:
I let vscode format the card-header the tabs are in to resolve the indentation. It looks like that lined everything up. Tooltips given ids as suggested.